Library/Formula/couchdb.rb

```diff
@@ -32,7 +38,7 @@ def install
     # in the welcome message
     inreplace "etc/couchdb/default.ini.tpl.in" do |s|
       s.gsub! "%package_author_name%", "Homebrew"
-      s.gsub! "%version%", "%version%-1"
+      s.gsub! "%version%", "%version%-3"
```

Comment:
done, produces `version = 1.6.1_3` which seems super reasonable & easy - thanks for the #protip @mikemcquaid